tests/components/twitch/test_sensor.py

```diff
@@ -80,6 +80,10 @@ async def test_oauth_with_sub(
     sensor_state = hass.states.get(ENTITY_ID)
     assert sensor_state.attributes["subscribed"] is True
     assert sensor_state.attributes["subscription_is_gifted"] is False
+    assert {"1000": 1, "2000": 2, "3000": 3}.get(
+        sensor_state.attributes["subscription_tier"], None
+    ) == 1
```

Comment:
The test is incorrect, that should just stay the same

Reply:
This value should be what the sensor attribute ends up as though, shouldn't it?  Or does it go through that part of `TwitchCoordinator`?

Reply:
the subscription tier is now what we put into the fixtures right?

Reply:
The fixture data (corrected a few minutes ago) is what gets returned from the API, so it's a string there.  The attribute `subscription_tier`, which is what gets displayed to the user, would be an integer.

Reply:
yes, then the test should be just `sensor_state.attributes["subscription_tier"] == 1`?